solana/bridge_token_factory/programs/bridge_token_factory/src/instructions/admin/initialize.rs

```diff
@@ -118,7 +119,8 @@ impl Initialize<'_> {
             },
             paused: 0,
             pausable_admin,
-            padding: [0; 67],
+            metadata_admin,
```

Comment:
Since the role management is more complex for Solana than Near, then we can later move all the management to Near, so we can sign messages with specific actions.